### PR TITLE
Add tolerations in values.yaml

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

File: charts/rancher-cis-benchmark/v0.0.1/values.yaml [hunk 1]
```diff
@@ -66,7 +68,8 @@ resources: {}
 
 nodeSelector: {}
 
-tolerations: []
+tolerations: 
+- operator: Exists
```
Comment (prachidamle) [inline charts/rancher-cis-benchmark/v0.0.1/values.yaml:72]: Why are we changing the kube-bench tolerations to match all @StrongMonkey?

Comment (StrongMonkey) [reply charts/rancher-cis-benchmark/v0.0.1/values.yaml:72]: We need to be able to schedule this pod to any host. In a setup with only etcd and controlplane nodes, CIS can't be run.

**Review by prachidamle — APPROVED** [review]
LGTM

**Review by leodotcloud — APPROVED** [review]
LGTM

**Pull request merged**